assets.md

```diff
@@ -2,29 +2,44 @@
 
 This file is intended to list all the assets controlled by OpenTelemetry.
 
-- Mailing list cncf-opentelemetry-net-maintainers@lists.cncf.io
-    - Used to register service accounts and as a NuGet OpenTelemetry organization e-mail. All .NET SIG maintainers are part of this list. Owned by CNCF.
-    - Admin: [add member](https://lists.cncf.io/g/cncf-opentelemetry-net-maintainers/).
+## CI pipelines
+
 - OpenTelemetry [Azure Pipeline](https://dev.azure.com/opentelemetry/).
```

Comment:
@SergeyKanzhelev this is CI, right?

Reply:
yes, but we switched to github actions. I can check and then remove